doc/tutorial_1.rst

```diff
@@ -498,24 +498,28 @@ Let's rewrite the ``imports`` task to use python action:
 
 
 
-Function ``get_imports`` is used as task's action.
-It returns a dictionary, this will be saved by ``doit`` in its internal database. The returned dictionary must contain only values that can be encoded in JSON.
+The function ``get_imports`` is used as the task's action.
+It returns a dictionary, which will be saved by ``doit`` in its internal database. The returned dictionary must contain only values that can be encoded as JSON.
 
-Note that the parameter ``module_path`` is passed into task definition of ``actions``.
-Instead of just specifying a callable it takes a tuple *(callable, args, kwargs)*.
+``get_imports`` takes the path's module as a parameter. The value of this parameter to be used upon task execution is specified in the ``actions`` of the task definition. Generally speaking, each element of the ``actions`` array is a tuple *(callable, args, kwargs)*.
```

Comment:
Can you still include the parameter name, like:

``get_imports`` takes the path's module as a parameter (``module_path``).